Makefile

```diff
@@ -36,10 +36,17 @@ TESTFLAGS    :=
 DUPLFLAGS    := -t 100
 
 ifeq ($(STATIC),1)
+# Static linking with glibc is a bad time. If we're building statically,
+# assume musl is installed and use that.
```

Comment:
I'm not sure why you bring up that it's linux specific - static builds don't work on OS X, anyway. I'm in favour of reworking this so it produces a useful error message in case this assumption is invalid, but I don't think removing it entirely buys us anything. If nothing else, this documents some hard-earned knowledge.